src/app.jsx

```diff
@@ -229,17 +222,15 @@ class Application extends React.Component {
                             [system ? "systemContainersLoaded" : "userContainersLoaded"]: true,
                         };
                     });
-                    if (init) {
-                        this.updateContainerStats(system);
-                        for (const container of reply || []) {
-                            this.inspectContainerDetail(container.Id, system);
-                        }
+                    this.updateContainerStats(system);
+                    for (const container of reply || []) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230705-104721-08f3c273-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)